src/main/java/org/mockbukkit/mockbukkit/block/data/AgeableDataMock.java

```diff
@@ -37,4 +47,11 @@ public int getMaximumAge()
 		return this.getLimitationValue(BlockDataLimitation.Type.MAX_AGE);
 	}
 
+	@Override
+	@SuppressWarnings({"java:S2975", "java:S1182"})
```

Comment:
>@SuppressWarnings({"java:S2975", "java:S1182"})

This suppresses 2 sonar errors:

- **[java:S2975](https://sonarcloud.io/organizations/mockbukkit/rules?open=java%3AS2975&rule_key=java%3AS2975)**:
> "clone" should not be overridden
We are using the copy constructor as suggested, but Bukkit requires the use of this method to clone.

- **[java:S1182](https://sonarcloud.io/organizations/mockbukkit/rules?open=java%3AS1182&rule_key=java:S1182)**:
> Classes that override "clone" should be "Cloneable" and call "super.clone()"
We will not call super.clone, since we delegate that to constructor.